tools/Replay.hpp

```diff
@@ -137,6 +151,26 @@ struct Stats {
     std::unordered_map<std::string, MessageStat> _recvd;
     std::unordered_map<std::string, MessageStat> _sent;
 
+    size_t GetMemoryUsage()
+    {
+        std::ifstream smapsFile("/proc/" + std::to_string(getpid()) + "/smaps");
+
+        std::string line;
+        size_t totalDirtyPss = 0;
+
+        while (std::getline(smapsFile, line)) {
```

Comment:
whatever the majority bracing style is in the rest of the file: we should use that - please check =)